### DISCUSSION TITLE
New Feature

### DISCUSSION THREAD

**Discussion by pratiktank2002** [damms005/devdb-vscode#54] [Ideas]
1. Can you add new search bar at top of the table listing, i have almost 25 to 30 table in my database so i have to scroll up and down too much.
2. For table filter , where you sawing null, can you add 1 button or on click of that `?` button we can filter raw that is not empty , or in other word `!= NULL` filter
![image](https://github.com/damms005/devdb-vscode/assets/126856504/f420664e-c9fb-4da8-9a2a-2cb3b865ee16)

**Comment by damms005** [comment]
These are nice suggestions!

Hopefully I can launch them with v2 because that is what I have on my plate for the project right now.

**Reply by damms005** [reply]
I should add that for your long tables list issue, you may find either of [context menu navigation](https://github.com/damms005/devdb-vscode/?tab=readme-ov-file#context-menu-entry) or the [code lens support](https://github.com/damms005/devdb-vscode/?tab=readme-ov-file#laravel-model-code-lens) helpful

**Comment by pratiktank2002** [comment]
Thanks for your reply, btw I am very impressed with your extension. Now I
don't have to go to the tab to see the database records.. 👍.

**Reply by damms005** [reply]
Glad you like it. v2 has very excitingly useful features.

Don't forget to share about the extension to friends and colleagues in your network too!

**Comment by damms005** [comment]
The search bar feature is now added. Enjoy!
